### ISSUE TITLE
Why nested TAG mounted 2 times?

### ISSUE THREAD

**Issue by ufocool** [riot/route#87]
```
<app>
  <router>
  <route path=“/demo”> <demo /> </route>
  </router>
</app>
```
```
<demo>
…
  this.on('mount', () => {
    console.log(“tag demo mount”);
  })
…
</demo>
```
Log console:

> tag demo mount
> tag demo mount

Why nested TAG mounted 2 times?
Why 2 records tag demo mount?

IF TAG <demo> insert before <router> then tag mounted 1 times...

**Comment by GianlucaGuarini** [comment]
It seems to be related to the new riot version 3.1.1 I will investigate it

**Comment by GianlucaGuarini** [comment]
@cognitom i need your help here I couldn't figure out where is the issue

**Comment by GianlucaGuarini** [comment]
Here an example demonstrating the issue http://plnkr.co/edit/8VI1J9BCRNvGVK5SLOLX?p=preview

**Comment by cognitom** [comment]
Will check soon. Wait a bit.

**Comment by cognitom** [comment]
Here's a plunker, I confirmed the double mounting @ufocool has reported.
http://plnkr.co/edit/ej93O9itH04HieD71w0r?p=preview

Hmm, I'll look it more deeply.

**Comment by cognitom** [comment]
Oops, @GianlucaGuarini already made an example.

**Comment by GianlucaGuarini** [comment]
fixed in riot@3.2.1
